src/ItemFlagMenu/ItemFlagMenu.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import List from '@material-ui/core/List';
+import { ListItem, ListItemText, makeStyles } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import { FLAG_LIST_MAX_HEIGHT } from '../constants';
+import { Flag } from '../types';
+
+export interface ItemFlagMenuProps {
+  flags: Flag[];
+  onFlag: any;
+  open: boolean;
+  setOpen: any;
+  selectedFlag: Flag;
+  setSelectedFlag: any;
```

Comment:
Could you try removing these `any` types? At worst go with `Function` and disable eslint.